Doc: Troubleshooting dashboard 404 (#6215)
dask/dask: docs/source/diagnostics-distributed.rst

forward port 8888 (the default Jupyter port) to port 8001 with
``ssh -L 8001:localhost:8888 user@remote``.

Required Packages
~~~~~~~~~~~~~~~~~

> If you are unable to see the dashboard (404 error), you might also be missing some of the

Reviewer: ```suggestion
Bokeh must be installed in your scheduler's environment to run the dashboard. If it's not the dashboard page will instruct you to install it.

Depending on your configuration, you might also need to install ``jupyter-server-proxy`` to access the dashboard.
```

> packages required to build it. Check that you have both ``bokeh`` and ``jupyter-server-proxy``

Reviewer 2: `jupyter-server-proxy` isn't required, but provides extra functionality. Perhaps:

```
Check that you have `bokeh` installed. Depending on your configuration,
you may also need `jupyter-server-proxy`.
```

forward port 8888 (the default Jupyter port) to port 8001 with
``ssh -L 8001:localhost:8888 user@remote``.

Required Packages
~~~~~~~~~~~~~~~~~

Bokeh must be installed in your scheduler's environment to run the dashboard. If it's not the dashboard page will instruct you to install it.

Depending on your configuration, you might also need to install ``jupyter-server-proxy`` to access the dashboard.

> installed. 

Reviewer: ```suggestion
```

forward port 8888 (the default Jupyter port) to port 8001 with
``ssh -L 8001:localhost:8888 user@remote``.

Required Packages
~~~~~~~~~~~~~~~~~

Bokeh must be installed in your scheduler's environment to run the dashboard. If it's not the dashboard page will instruct you to install it.

Depending on your configuration, you might also need to install ``jupyter-server-proxy`` to access the dashboard.

> packages required to build it. Check that you have both ``bokeh`` and ``jupyter-server-proxy``

Reviewer: ```suggestion
```
